### PR TITLE
David bradford

### PR DESCRIPTION
**Pull request by deviator53**
- created a change password page
- did a cleanup of password reset, email verified and route to match with backend url

### CODE REVIEW

File: src/pages/ForgotPassword.jsx [hunk 1]
```diff
@@ -37,21 +37,18 @@ const ForgotPassword = () => {
...
       console.log(error);
-      if (
-        error.response &&
-				error.response.status >= 400 &&
-				error.response.status <= 500
-      ){
         setError(error.response.data.message);
-				setMsg("");
+				setEmail("");
+        setTimeout(() => {
```
Comment (PhilemonBrain) [inline src/pages/ForgotPassword.jsx:46]: why this setTimeout?

File: src/pages/PasswordReset.jsx [hunk 2]
```diff
@@ -102,27 +102,20 @@ const PasswordReset = () => {
 			// Post to server
 			
 			try {
-				const res = await axios.post(url, { password });
+				const res = await axios.post(url, { password, passwordConfirm });
```
Comment (PhilemonBrain) [inline src/pages/PasswordReset.jsx:105]: we have a fetch hook. try and use it

File: src/pages/PasswordReset.jsx [hunk 3]
```diff
@@ -102,27 +102,20 @@ const PasswordReset = () => {
...
-				  ){
-					setError(error.response.data.message);
-					setTimeout(()=> {
-						setError("");
-					},5000);
-					setMsg("");
-				  }
+				
+				setError(error.response.data.message);
+				setTimeout(()=> {
```
Comment (PhilemonBrain) [inline src/pages/PasswordReset.jsx:113]: why setTimeout? what are u trying to achieve

File: src/App.js [hunk 4]
```diff
@@ -68,7 +69,9 @@ const App = () => {
           <Route path='/signup' element={<Signup />} />
           <Route path='/forgot-password' element={<ForgotPassword />} />
           <Route path='/email-verify' element={<EmailVerify />} />
-          <Route path='/password-reset/:id/:token' element={<PasswordReset />} />
+          <Route path='/password-reset/:token' element={<PasswordReset />} />
+          <Route path='/change-password' element={<ChangePassword />} />
```
Comment (PhilemonBrain) [inline src/App.js:73]: where did u handle the password strength check?

**Review by PhilemonBrain — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 5]
```diff
@@ -8,6 +8,7 @@ import Cookies from 'universal-cookie'
 import { ForgotPassword, Home, LoginPage, SingleApi, UserProfile, Categories, Category, CreateOrg, Signup, Settings, MyApiPage, Endpoint } from './pages'
 import EmailVerify from './pages/EmailVerify';
 import PasswordReset from './pages/PasswordReset'
+import ChangePassword from './pages/ChangePassword'
```
Comment (taiwoAkindele) [inline src/App.js:11]: add the EmailVerify and other pages you worked on to the index.js in the pages folder and export them from there.

Comment (taiwoAkindele) [inline src/App.js:11]: add the ChangePassword and other pages you worked on to the index.js in the pages folder and export them from there.

File: src/pages/ChangePassword.jsx [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    '@media screen and (max-width: 800px)': {
+      width: '90%'
+    },
+    '@media screen and (min-width: 1270px)': {
+      width: '60%'
+    }
+  }
+})
+
+const ChangePassword = () => {
```
Comment (taiwoAkindele) [inline src/pages/ChangePassword.jsx:25]: I don't think this is working yet or is it? @deviator53

**Review by taiwoAkindele — APPROVED** [review]

**Review by PhilemonBrain — APPROVED** [review]

**Pull request merged**